src/store/messagesStore.js

```diff
@@ -529,11 +529,16 @@ const actions = {
 			&& (message.systemMessage === 'message_deleted'
 				|| message.systemMessage === 'reaction'
 				|| message.systemMessage === 'reaction_deleted'
-				|| message.systemMessage === 'reaction_revoked')) {
+				|| message.systemMessage === 'reaction_revoked'
+				|| message.systemMessage === 'message_edited')) {
 			// If parent message is presented in store already, we update it
 			const parentInStore = context.getters.message(token, message.parent.id)
 			if (Object.keys(parentInStore).length !== 0) {
 				context.commit('addMessage', { token, message: message.parent })
+				if (message.systemMessage === 'message_edited') {
+					// End of process for edited messages
```

Comment:
```suggestion
					EventBus.$emit('message-edited')
```

This will trigger it for your messages and for external ones